From 69a63f208e7070111d6436d4b2fa87053ae5ca8f Mon Sep 17 00:00:00 2001
From: Jasper St. Pierre <jstpierre@mecheye.net>
Date: Wed, 16 May 2012 15:28:05 +0000
Subject: notificationDaemon: Fix copy-paste typo

Since this line was introduced in 7458d3e, "Approve file transfer channels",
it's quite obvious that the line was meant to filter out transfer channel
notifications.

https://bugzilla.gnome.org/show_bug.cgi?id=676175
---
diff --git a/js/ui/notificationDaemon.js b/js/ui/notificationDaemon.js
index c6a941f..4a413ce 100644
--- a/js/ui/notificationDaemon.js
+++ b/js/ui/notificationDaemon.js
@@ -233,7 +233,7 @@ const NotificationDaemon = new Lang.Class({
         if (appName == 'Empathy' && (hints['category'] == 'im.received' ||
               hints['category'] == 'x-empathy.im.room-invitation' ||
               hints['category'] == 'x-empathy.call.incoming' ||
-              hints['category'] == 'x-empathy.call.incoming"' ||
+              hints['category'] == 'x-empathy.transfer.incoming' ||
               hints['category'] == 'x-empathy.im.subscription-request' ||
               hints['category'] == 'presence.online' ||
               hints['category'] == 'presence.offline')) {
--
cgit v0.9.0.2
